src/production/transformation/__tests__/__snapshots__/condition.unwrapper.test.ts.shapshot.json

```diff
@@ -1,50 +1,42 @@
 {
-  "554399d5": "var a = 1;\nconsole.log(\"yes\");\nvar b = 2;\n    ",
-  "52627dbc": "var a = 1;\nconsole.log(\"no\");\nvar b = 2;\n    ",
-  "4946215e": "alert(1)",
-  "6c233153": "alert(1)",
-  "03dc305bf": "alert(1)",
-  "0492d3179": "alert(1)",
-  "059ef9704": "alert(1)",
-  "27e7d078": "alert(1)",
-  "6d7337eb": "alert(2)",
-  "5589dc60": "alert(2)",
-  "062cfc36c": "",
-  "055ff0486": "alert(2)",
-  "2fbbc16f": "alert(2)",
-  "52689aa5": "",
-  "06c691f96": "alert(2)",
-  "0518e41a1": "alert(2)",
-  "0133ab04b": "",
-  "049660f05": "alert(2)",
-  "0526c8290": "alert(2)",
-  "4e6a2984": "",
-  "03baccc54": "alert(1)",
-  "061eec15f": "alert(1)",
-  "58621d33": "alert(1)",
-  "06297fa07": "alert(1)",
-  "04e1bfa92": "alert(1)",
-  "56670bc6": "alert(1)",
-  "1d905949": "alert(1)",
-  "761bb947": "alert(1)",
-  "19d09500": "alert(1)",
-  "030924490": "alert(1)",
-  "402bb71c": "alert(2)",
-  "188138d4": "",
-  "02b857c53": "alert(2)",
-  "5fb0ff63": "",
-  "50a020bd": "alert(2)",
-  "c231453": "",
-  "48b9bc0c": "alert(2)",
-  "05ae3e1c": "",
-  "07d115ec5": "alert(1)",
-  "079b5be6b": "alert(1)",
-  "603b4b4e": "alert(1)",
-  "03aefc31e": "alert(1)",
-  "0e5deb38": "console.log(\"yes\")",
-  "41d36129": "alert('no')",
-  "573e3511": "alert('no')",
-  "07044b600": "console.log(\"yes\")",
-  "f91ebe7": "function test(){\n          return;\n      }\n      ",
-  "050d94879": "function test(){\n          console.log(1)\n        }\n    "
+  "3836108c": "var a = 1;\nconsole.log(\"yes\");\nvar b = 2;\n    ",
```

Comment:
Could you revert these snapshots?

Reply:
Are they not being used? They are generated when I run `npm test`.

Reply:
I am not sure why the hashes are changing ....